### ISSUE TITLE
Uncaught TypeError: Cannot add property useHistoryCustom, object is not extensible

### ISSUE THREAD

**Issue by xuatz** [supertokens/supertokens-auth-react#366]
## Description

It might be due to my setup, so more context about the situation.

First it appears that because I'm using vite, i cannot use `require`, while there might be workarounds that I'm not aware of, let's move on.

As a result, the alternative approach i went with was (btw im also using ts for my project):
```ts
import * as ReactRouterDom from 'react-router-dom'
...
	<div>
		{getSuperTokensRoutesForReactRouterDom(ReactRouterDom)}
	</div>
...
```

And with that, when i try to navigate to the root route, i got this error

```
Uncaught TypeError: Cannot add property useHistoryCustom, object is not extensible
    at SuperTokens2.getSuperTokensRoutesForReactRouterDom (:3000/node_modules/.vite/chunk-NPCAOA46.js?v=29f04396:243:56)
    at Router (:3000/src/Router.tsx?t=1642829627830:44:16)
    at renderWithHooks (:3000/node_modules/.vite/chunk-CJMVPCGN.js?v=29f04396:11061:26)
    at mountIndeterminateComponent (:3000/node_modules/.vite/chunk-CJMVPCGN.js?v=29f04396:13181:21)
    at beginWork (:3000/node_modules/.vite/chunk-CJMVPCGN.js?v=29f04396:13968:22)
    at HTMLUnknownElement.callCallback2 (:3000/node_modules/.vite/chunk-CJMVPCGN.js?v=29f04396:3671:22)
    at Object.invokeGuardedCallbackDev (:3000/node_modules/.vite/chunk-CJMVPCGN.js?v=29f04396:3696:24)
    at invokeGuardedCallback (:3000/node_modules/.vite/chunk-CJMVPCGN.js?v=29f04396:3730:39)
    at beginWork$1 (:3000/node_modules/.vite/chunk-CJMVPCGN.js?v=29f04396:17077:15)
    at performUnitOfWork (:3000/node_modules/.vite/chunk-CJMVPCGN.js?v=29f04396:16305:20)
 ```

That's when I looked into the code and still see some usage of `useHistoryCustom`.

Let me know if you know what's wrong, thanks!

**Comment by rishabhpoddar** [comment]
This issue should have been fixed now in v0.18.2. I could not replicate it locally though, but I did try and fix it anyway.

If this issue still persists, please feel free to reopen :)

**Comment by xuatz** [comment]
I verified that the fix worked, thanks!
